cherryontech/unmatched
# Copy/Maggie & James' bio pages

## Description
This PR adds copy to Maggie & James' bios, completing their bio pages. 


## Related Tickets & Documents
This was done in support of our design team, as they needed help with the copy.


## Acceptance Criteria



## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|     | :sparkles: New feature     |
|  ✓  | ✏️  Copy     |
|     | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Mobile & Desktop Screenshots/Recordings
<img width="1253" alt="Screenshot 2023-11-14 at 11 34 07 AM" src="https://github.com/cherryontech/unmatched/assets/115492619/ea3354a2-8c1b-4e6c-b7e7-6cd8ac6be91b">



## What I learned
Since `fullDataSet` is stored once as a `useRef` when Vite is initially run, you need to re-run Vite to see changes made to `bioData.js`

## Testing steps

1. Switch to this branch with the following command: `git switch feature/maggie-bio-page`
2. Run the following scripts: `npm run dev` to start Vite's dev build & `npm run tailwind` to start the Tailwind CSS build
3. View the app in browser (the `npm run dev` script will return the url you can see the app at)
4. Navigate to the the Resources page and then to Maggie's bio page and James' bio page. 
5. On each bio page, read through the copy, check for typos, makes suggestions, and check the links to make sure they work.

## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|  ✓ | 🙅 no documentation needed    |


## What gif best describes this PR or how it makes you feel?
![yay](https://media.giphy.com/media/MeIucAjPKoA120R7sN/giphy.gif)
